bindings/python/Makefile.am

```diff
@@ -33,7 +33,7 @@ EXTRA_DIST = \
 if ENABLE_PYTHON_BINDING
 
 all-local:
-	$(PYTHON) $(srcdir)/setup.py build
+	CXXFLAGS="$(CXXFLAGS)" $(PYTHON) $(srcdir)/setup.py build
```

Comment:
Or should I write 
`CXX="$(CXX)" CXXFLAGS="$(CXXFLAGS)" $(PYTHON) $(srcdir)/setup.py build`
without hardcoding gcc specfifics at CXXFLAGS